packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -66,51 +66,35 @@ const validateOperationTask = (task, props) => {
   })
 }
 
-// Note: `bypassVdiChainsCheck` must be enabled because the XAPI might be not
-// able to coalesce VDIs as fast as the tests run.
-//
-// See https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection
 describe('backupNg', () => {
-  let defaultBackupNg
-
-  beforeAll(() => {
-    defaultBackupNg = {
-      name: 'default-backupNg',
-      mode: 'full',
-      vms: {
-        id: config.vms.default,
-      },
-      settings: {
-        '': {
-          reportWhen: 'never',
-        },
-      },
-    }
-  })
-
   describe('.createJob() :', () => {
     it('creates a new backup job without schedules', async () => {
-      const backupNg = await xo.createTempBackupNgJob(defaultBackupNg)
+      const vms = {
+        id: config.vms.default,
```

Comment:
Why? I thought it was the default value?

Reply:
Needed for this test https://github.com/vatesfr/xen-orchestra/pull/4544/files/8a5a5a5cf107cf8501c9108506657b7ecff0d41d#diff-59a6717e72428c99e66f7baf3e1f93deR81